licenses/README.rst

```diff
@@ -1,5 +1,6 @@
 Licenses
 ========
 
-This directory holds license and credit information for astropy, works astropy is derived from, and/or datasets.
+This directory holds license and credit information for works astropy is derived from or distributes, and/or datasets.
```

Comment:
@bsipocz , I guess it is fine to leave the comma in now. Usually Oxford comma only applies to a sentence with 3 or more separate items, no? For example, "I like stars, planets, and dust." For 2 items, it would be, "I like stars and planets." (No comma there.)